scripts/run-vscode-integration-tests.js

```diff
@@ -0,0 +1,16 @@
+const shell = require('shelljs');
+shell.set('-e');
+shell.set('+v');
+
+const path = require('path');
+const cwd = process.cwd();
+
+// Executes the tests in the out/test/vscode-integration directory
+shell.exec(
+  `cross-env CODE_TESTS_PATH='${path.join(
```

Comment:
@lcampos - I was wrong when I mentioned to you in Bellevue that the `vscode/bin/test` script necessarily runs all tests inside of the `out/test` directory. If I set the CODE_TESTS_PATH environment variable, I can override that default, and scope the tests being run to the `out/test/vscode-integration` directory instead. This is great because we can now have unit tests, integration tests, and vscode-integration test all inside of the `test` directory and still be able to run them separately.

Reply:
Very cool 👍